client/js/components/StatsScoringDialog.vue

```diff
@@ -50,9 +50,10 @@
           </v-list-item>
         </v-list>
         <p>
-          You can view the statistics and rankings for each season on this page. At the end of each
-          season, the top 4 players compete in a double elimination championship tournament. Do you
-          have what it takes to become
+          You can view the statistics and rankings for each season on
+          <a href="#/stats" target="_self"> the Stats page </a> . At the end of each season, the top
```

Comment:
Does this work nicely? Normally we use vue's `<router-link>` component for internal SPA routing. @seriouslysean got an opinion on this?

Reply:
Yep, think so!

https://user-images.githubusercontent.com/1594170/195737801-3adaaf64-da0d-4bc2-b2f3-cb55146e62e6.mp4

I can change it if you'd like

Reply:
I would switch this. It'll work with the hash router but would force a hard reload if we ever switch to the web router, https://router.vuejs.org/guide/essentials/history-mode.html#hash-mode.

Reply:
Yeah seconded. @byarmis can we update this to a `<router-link>`that uses the route `name` to navigate to the `Stats` page? That'll be more flexible if we change the router mode or the page url

Reply:
Sure, no problem.

Done!